packages/components/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { User } from '@codesandbox/common/lib/types';
+import styled from 'styled-components';
+import { Element } from '../Element';
+import { Text } from '../Text';
+
+interface IAvatarProps {
+  user: User;
+}
+
+export const AvatarImage = styled.img(
+  css({
+    width: 8,
+    height: 8,
+    borderRadius: 'small',
+    border: '1px solid rgba(255, 255, 255, 0.46)',
+    backgroundColor: 'rgba(255, 255, 255, 0.46)', // fallback for loading image
+  })
+);
+
+export const Pro = styled(Text).attrs({ size: 1, weight: 'bold' })(
+  css({
+    backgroundColor: 'blues.700',
+    color: 'sideBarTitle.foreground',
+    borderRadius: 'small',
+    paddingX: 2,
+    paddingY: '2px', // no 2 in spaces
+    border: '1px solid',
+    borderColor: 'sideBar.border',
+    transform: 'translateX(-50%) translateY(-100%);',
+    position: 'absolute',
+    top: '100%',
+  })
+);
+
+export const Avatar = ({ user }: IAvatarProps) => (
+  <Element style={{ position: 'relative' }}>
```

Comment:
If you want, you can also make this `<Element css={{ position: 'relative' }}>`

Reply:
Doesn't work :(

Reply:
oh my bad. I thought it did. It might be useful to have, I can work on that tomorrow.